docs/automatic-ofed-upgrade.md

```diff
@@ -35,7 +35,7 @@ spec:
         # specify a label selector to filter pods on the node that need to be drained
         podSelector: ""
         # specify the length of time in seconds to wait before giving up drain, zero means infinite
```

Comment:
nit you should mention in this doc that the default timeout if not specified is 5 minutes